addons/sale/models/sales_team.py

```diff
@@ -1,65 +1,58 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import calendar
 from datetime import date
-from dateutil import relativedelta
-import json
 
-from openerp import tools
-from openerp.osv import fields, osv
-from openerp.tools.float_utils import float_repr
+from odoo import api, fields, models
 
-class crm_team(osv.Model):
-    _inherit = 'crm.team'
 
-    def _get_sales_to_invoice_amount(self, cr, uid, ids, field_name, arg, context=None):
-        obj = self.pool['sale.order']
+class CrmTeam(models.Model):
+    _inherit = 'crm.team'
 
-        res = dict.fromkeys(ids, 0)
+    use_quotations = fields.Boolean('Quotations', default=True,
+        help="Check this box to manage quotations in this sales team.")
+    use_invoices = fields.Boolean('Invoices',
+        help="Check this box to manage invoices in this sales team.")
+    invoiced = fields.Integer(
+        compute='_compute_invoiced',
+        string='Invoiced This Month', readonly=True,
+        help="Invoice revenue for the current month. This is the amount the sales "
+                "team has invoiced this month. It is used to compute the progression ratio "
+                "of the current and target revenue on the kanban view.")
+    invoiced_target = fields.Integer(
+        string='Invoice Target',
+        help="Target of invoice revenue for the current month. This is the amount the sales "
+             "team estimates to be able to invoice this month.")
+    sales_to_invoice_amount = fields.Integer(
+        compute='_compute_sales_to_invoice_amount',
+        string='Amount of sales to invoice', readonly=True,
+    )
+    currency_id = fields.Many2one("res.currency", related='company_id.currency_id',
+        string="Currency", readonly=True, required=True)
+
+    @api.multi
+    def _compute_sales_to_invoice_amount(self):
         domain = [
-            ('team_id', 'in', ids),
+            ('team_id', '=', self.ids),
             ('invoice_status', '=', 'to invoice'),
         ]
-        amounts = obj.read_group(cr, uid, domain, ['amount_total', 'team_id'], ['team_id'], context=context)
+        amounts = self.env['sale.order'].read_group(domain, ['amount_total', 'team_id'], ['team_id'])
```

Comment:
Self is crm_team records and not sale_order. It groups the sale_order by crm_team and then set the sales_to_invoice_amount for each crm_team.